### DISCUSSION TITLE
Flow control - Show

### DISCUSSION THREAD

**Discussion by HuJean** [solidjs/solid#1057] [Q&A]
The condition result is always false?

![image](https://user-images.githubusercontent.com/7037477/173477754-54662178-1991-4cb8-b6cd-bd0da62a33a0.png)

**Comment by ryansolid** [comment]
I don't think so. We also support:
```js
<Show when={user()}>{
  (user) => <div>{user.name}</div>
}</Show>
```
This serves as a form of keying to each user.
